$NetBSD$

Fix SDL 2 detection on some platforms with more portable test(1)
invocation.

--- configure.ac.orig	2024-02-04 22:00:47.864879289 +0000
+++ configure.ac
@@ -1219,7 +1219,7 @@ if test x$fim_handle_sdl != xno; then
                 LIBS="$LIBS `$LIBSDL_CONFIG $LIBSDL_CONFIG_FLAGS`"
 		# finish this !
                 #FIM_LIB_OBJECTS="$FIM_LIB_OBJECTS SDLDevice.o"
-		if test x`$LIBSDL_CONFIG --version | head -c 1` == x2; then
+		if test x`$LIBSDL_CONFIG --version | head -c 1` = x2; then
 			AC_DEFINE(FIM_WITH_LIBSDL, 2, [Defined if libsdl support is enabled, and set to 1 or 2.])
 		else
 			AC_DEFINE(FIM_WITH_LIBSDL, 1, [Defined if libsdl support is enabled, and set to 1 or 2.])
